### PR TITLE
[JS] Add quiz app with timer

### LINKED ISSUE #126: Quiz App with Timer
A Quiz App with Timer using HTML,CSS,JS.

This is a quiz app with timer to create a more challenging environment.

ezgif-4-d6fe98f8b3

ezgif-4-c06dbb3269

ezgif-4-4f551ac1c5

I would really appreciate if u could allow me to do the project

### PR DESCRIPTION
**Pull request by Vivek-GuptaXCode**
# Title and Issue number 

Title : Quiz App with Timer

Issue No. : #126 

Code Stack : 

Close #126 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/helpers/challenges.js [hunk 1]
```diff
@@ -450,6 +450,13 @@ export const challenges = [
     developer: 'sadanandpai',
     tags: [],
   },
+  {
+    title: 'Quiz App with Timer',
+    link: 'Quiz-app-with-timer',
```
Comment (sadanandpai) [inline native/helpers/challenges.js:455]: please do not use any capital letters

rename the folder as along

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: native/helpers/challenges.js (hunk 1 above)
Comment (Vivek-GuptaXCode) [reply native/helpers/challenges.js:455]: Okay got it

File: native/machine-coding/Quiz-app-with-timer/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MCQ Quiz</title>
+    <!-- Stylesheet -->
+    <link rel="stylesheet" href="index.css" />
+    <script src="../../helpers/navbar.js" type="module"></script>
```
Comment (arpansaha13) [inline native/machine-coding/Quiz-app-with-timer/index.html]: Replace this script with this one `../../helpers/header.js`

**Review by arpansaha13 — CHANGES_REQUESTED** [review]

**Comment by arpansaha13** [comment]
Folder name should be in kebab case `quiz-app-with-timer`

**Review by arpansaha13 — COMMENTED** [review]
image

Looks like there is a problem with the folders
Please fix these

**Comment by Vivek-GuptaXCode** [comment]
Should i create a new pr?

**Comment by arpansaha13** [comment]
No dont create new PR please

**Comment by Vivek-GuptaXCode** [comment]
Sorry for the inconvenience but i really messed up with my pr so in order to ensure a smooth workflow i humbly ask you to close this PR so that i can submit it again with proper changes as mentioned here

**Comment by arpansaha13** [comment]
Please check if it is running fine in your local
If everything is working fine then let us know
:)

**Review by arpansaha13 — APPROVED** [review]

**Comment by Vivek-GuptaXCode** [comment]
Thanks for the help,i would make sure to gain some valuable insight from this first PR and learn more.Sorry for the inconvenience.

**Comment by arpansaha13** [comment]
Its okay. And thanks for the contribution

**Pull request merged**